docs/sql-reference/sql-statements/data-manipulation/UPDATE.md

```diff
@@ -2,11 +2,11 @@
 
 Updates rows in a Primary Key table.
 
-In versions earlier than version 3.0, the UPDATE statement only supports single-table UPDATE and does not support common table expressions (CTEs). Starting from version 3.0, StarRocks enriches the syntax to support multi-table joins and CTEs. If you need to join the table to be updated with other tables in the database, you can reference these other tables in the FROM clause or CTE.
+In versions earlier than version 3.0, the UPDATE statement only supports single-table UPDATE and does not support common table expressions (CTEs). Starting from version 3.0, StarRocks enriches the syntax to support multi-table joins and CTEs. If you need to join the table to be updated with other tables in the database, you can reference these other tables in the FROM clause or CTE. Since version 3.1, the UPDATE statement supports the column mode, which are suitable for scenarios involving a small number of columns but a large number of rows, resulting in faster update speeds.
```

Comment:
readers cannot understand the term “column mode”, change to "column-mode partial update", which is suitable

Reply:
add privilege requirements for update sql, for example, "This command requires the UPDATE privilege on the table you want to update."